external/dynamo/generalized/frontend.py

```
@@ -0,0 +1,445 @@
+        config = uvicorn.Config(self.app, host=host, port=port, log_level="info")
+        server = uvicorn.Server(config)
+        logging.info("Starting FastAPI server on %s:%s", host, port)
+        await server.serve()
+
+    # ----------------- throughput helpers -----------------
+    async def _inc_tps(self):
+        try:
+            async with self._tps_lock:
+                self._tps_count += 1
+        except Exception:
+            pass
```

Ruff (0.14.8)

`try`-`except`-`pass` detected, consider logging the exception

(S110)